docs/quickstart.md

```diff
@@ -0,0 +1,195 @@
+
+!!! note
+    If preferred, you can directly jump to the Getting Started guides of the popular hardware platforms:
+
+    * [Nordic](LwM2M_Client/Nordic/Thingy91/)
+    * [STMicroelectronics](LwM2M_Client/STMicroelectronics/Building_LwM2M_applications/Building_LwM2M_applications_with_I-CUBE-Anjay/), 
+    * [ESP32](LwM2M_Client/ESP32/M5Stick/) 
+    * [Raspberry Pi](Anjay_integrations/RaspberryPi_integration/RaspberryPi-Pico-W/)
+
+
+!!! Info
+    For more information about the Anjay LwM2M Client, visit the <a href="https://avsystem.github.io/Anjay-doc/index.html" target="_blank">**Anjay Library Documentation**</a>.
+
+
+### Compile the Anjay demo client:
+
+=== "Linux"
+    Open up your command line interface and install the required dependencies:
+
+    ```
+    sudo apt-get install git build-essential cmake libmbedtls-dev zlib1g-dev
+    ```
+
+    Create a new directory, e.g. called `AVSystem`
+
+    ```
+    mkdir ~/AVSystem \
+        && cd AVSystem
+    ```
+
+    Clone the Anjay GitHub repository and compile the Anjay demo client using Cmake.
+
+    ```
+    git clone https://github.com/AVSystem/Anjay.git \
+        && cd Anjay \
+        && git submodule update --init \
+        && cmake . \
+        && make -j
+    ```
+
+=== "Mac"
+    Open up your terminal and install the required dependencies using [Homebrew](https://brew.sh/).
+
+    ```
+    brew install cmake mbedtls
+    ```
+
+    Create a new directory, e.g. called `AVSystem`
+
+    ```
+    mkdir ~/AVSystem \
+        && cd AVSystem
+    ```
+
+    Clone the Anjay GitHub repository and compile the Anjay demo client using Cmake.
+
+    ```
+    git clone https://github.com/AVSystem/Anjay.git \
+        && cd Anjay \
+        && git submodule update --init \
+        && cmake . \
+        && make -j
+    ```
+
+=== "Windows"
+    **Install the required dependencies**
+
+    1. Install [MSYS2](http://www.msys2.org/)
+    
+    1. Install [Git for Windows](https://gitforwindows.org/)
+    
+        !!! Note 
+            You can also install these using [Chocolatey](https://chocolatey.org/): `choco install git msys2` but make sure to follow the instructions to update MSYS2 after installing it.
+    
+    1. Open the appropriate MINGW shell (e.g., `C:\msys64\mingw32.exe` or `C:\msys64\mingw64.exe`, depending on whether you want to build 32- or 64-bit binaries) and install the compile-time dependencies:
+
+        ```
+        pacman -Sy make ${MINGW_PACKAGE_PREFIX}-gcc ${MINGW_PACKAGE_PREFIX}-cmake ${MINGW_PACKAGE_PREFIX}-mbedtls
+        ```
+
+    **Compile the project**
+
+    Run the following commands in the MINGW shell, after navigating to the directory created using Git above:
+
+    ```
+    cmake -G"MSYS Makefiles" -DDTLS_BACKEND="mbedtls" .
+    make
+    ```
+
+## Add a new device in Coiote IoT DM
+
+After compiling the demo project, go back to the [Coiote IoT DM portal](https://eu.iot.avsystem.cloud/).
+
+Click the button **+ Add device** in the top-right corner.
```

Comment:
`Click **+ Add device** in the top-right corner.` it's better to avoid the word `button` in such cases